app/src/main/java/com/github/sdp/mediato/utility/adapters/CollectionAdapter.java

```diff
@@ -0,0 +1,97 @@
+package com.github.sdp.mediato.utility.adapters;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+import com.github.sdp.mediato.R;
+import com.github.sdp.mediato.errorCheck.Preconditions;
+import com.github.sdp.mediato.model.Review;
+import com.github.sdp.mediato.model.media.Collection;
+import com.github.sdp.mediato.model.media.Media;
+import java.util.List;
+
+public class CollectionAdapter extends RecyclerView.Adapter<CollectionAdapter.ViewHolder> {
+
+  private Context context;
+  private List<Review> reviews;
+  private Collection collection;
+
+
+  public CollectionAdapter(Context context, Collection collection) {
+    this.context = context;
+    this.collection = collection;
+    this.reviews = collection.getReviews();

```

Comment:
Small comment: this will be modified in a new PR since the reviews attribute in collection will have to change as a map.